Simplenote/Classes/SPCardViewController.swift

```diff
@@ -35,6 +38,85 @@ final class SPCardViewController: UIViewController {
     }
 }
 
+// MARK: - Presentation
+//
+extension SPCardViewController {
+
+    /// Presents card from another view controller animated
+    ///
+    func present(from viewController: UIViewController) {
+        viewController.addChild(self)
+
+        view.translatesAutoresizingMaskIntoConstraints = false
+        guard let containerView = viewController.view else {
+            fatalError("Can't load view")
+        }
```

Comment:
Nipticky: since this whole method depends on the guard being successful, perhaps that should be the very first thing we run?

Reply:
Right now we first add view controller as a child and then check / add view. 
If we move guard to the top, we would first access view (which triggers viewDidLoad) and only after that add view controller as a child.
My concern here that if for some reason we will want to access `parent` view controller from viewDidLoad, it won't work in the second case.